src/Testing/Commands/MakeExpectationCommand.php

```diff
@@ -61,12 +70,45 @@ public function handle(
 
         $basePath = $getBasePathAction->execute();
 
-        $inputClass = $this->getInputClass($basePath, $filesystem);
+        /** @phpstan-var class-string|string $class */
+        $class = (string) $this->input->getArgument('class');
+
+        if ($class === 'all') {
+            $finder = Finder::create()->files()->name('*.php');
```

Comment:
Hmm asi bych udělal factory které mě sestaví ten finder místo setup, připadá mě to čistější a lépe testovatelné stubem